dspace-api/src/main/java/org/dspace/app/util/GoogleMetadata.java

```diff
@@ -1055,6 +1054,7 @@ protected Bitstream findLinkableFulltext(Item item) throws SQLException {
 		List<Bundle> contentBundles = itemService.getBundles(item, "ORIGINAL");
 		for (Bundle bundle : contentBundles) {
             List<Bitstream> bitstreams = bundle.getBitstreams();
+            Collections.sort(bitstreams, googleBitstreamComparator);
```

Comment:
I have done a quick test. The choice of the file to display works as advertised but after visiting item view page the bitstream order is updated in the database. It appears Hibernate is updating the bitstream order after the sorting made by Collections.sort(bitstreams, googleBitstreamComparator). It seems similar to the issue described here: 

https://jira.duraspace.org/browse/DS-3599

Creating a new ArrayList avoids the issue (although there could be a better way to solve it):

	List<Bitstream> bitstreams = new ArrayList(bundle.getBitstreams());